DNA_RNA.py

```diff
@@ -0,0 +1,69 @@
+import string
+
+__nucleic_acid_seq_DNA__ = set('ACGT')
+__nucleic_acid_seq_RNA__ = set('AGTU')
```

Comment:
```suggestion
__nucleic_acid_seq_RNA__ = set('GUACguac')
```
the nitrogenous bases of RNA are guanine, uracil, adenine, and cytosine